src/bp/diag/utils.ts

```diff
@@ -62,7 +62,7 @@ export const testWebsiteAccess = async (label: string, url: string) => {
   const start = Date.now()
   let ip
   try {
-    ip = await dnsLookup(new URL(url).host)
+    ip = await dnsLookup(new URL(url).hostname)
```

Comment:
DNS lookup throws when we use `host` (which is of format `hostname:port`, e.g. `google:8080`).
We want to use `hostname` instead.